4001-5000/LIT4872MaqdemaWangel.xml

```diff
@@ -52,31 +52,92 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+              እመሕያው፡ ልጹቃን፡ ኀበ፡ ፈጣሪሆሙ፡ ይግበሩ፡ ምግባራቲሁ፡ ወይረሱ፡ መንግሥቶ።</ab></div>
+            </div>
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter2">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter2"><ab>ወካልኡ፡ ብቍዔት፡</ab></div>
+              <div type="textpart" subtype="incipit"><ab>ወበቍዔተ፡ ዝንቱ፡ መጽሓፍ፡ ብጹሕ፡ ካዕበ፡ እስመ፡ ውእቱ፡ ዘይትንበብ፡
+              እምኔሁ፡ ወይጤየቅ፡ ግብረ፡ መድኃኒቱ፡ ለአዳም፡ ወለዘርኡ፡ እምፂዋዌ፡ ሰይጣን።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወናሁ፡ ጸንዓ፡ በቍዔተ፡ ዝንቱ፡ መጽሐፍ፡ ንጹሕ፡ ወብዙኅ፡ ወባሕቱ፡
+              ተኃሥሦ፡ መፍቅድ፡ ኀበ፡ አንብቦቱ፡ ለዘኮነ፡ ልብወ፡ ወማእምረ።</ab></div>
+            </div>
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter3">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter3"><ab>ወሣልስ፡ ሥርዓት፡</ab></div>
+              <div type="textpart" subtype="incipit"><ab>ወሥርዓተ፡ ዝንቱ፡ መጽሐፍ፡ ክቡር፡ ወይእቲኬ፡ አንብቦቱ፡ ኵሎ፡ ጊዜ፡
+              ወኀዲገ፡ ሕሊናት፡ ለዓቂበ፡ አቅማሪሁ፡ ወተአምራቲሁ፡ ወትእኅዘ፡ ቦቱ።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወናሁ፡ ተጠየቀት፡ ሥርዓት፡ ወጸንዓት፡ ንብረቱ።</ab></div>
+            </div>
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter4">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter4"><ab>ወራቡዕ፡ ትእምርት፡</ab></div>
+              <div type="textpart" subtype="incipit"><ab>ወትእምርተ፡ ዝንቱ፡ መጽሐፍ፡ ክቡር፡ ወንጌል፡ ወወንጌል፡ ብሂል፡ በልሳነ፡
+              ፅርዕ፡ ወትርጓሜሁ፡ ስብከት፡ እንተ፡ ወሰከ፡ ውእቱ።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወናሁ፡ ተበየነ፡ ትምህርቱ። ወምክንያተ፡ ስሙ።</ab></div>
+            </div>
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter5">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter5"><ab>ወኃምስ፡ እምአይቴ፡</ab></div>
+              <div type="textpart" subtype="incipit"><ab>ወመሠረተ፡ ዝንቱ፡ መጽሓፍ፡ ክቡር፡ በኀበ፡ እግዚኣብሔር፡ ዓቢይ፡ ወዓዚዝ።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወናሁ፡ ነበረት፡ መሠረቱ፡ ኀበ፡ ላዕሉ፡ ወጸሓፍቲሁ።</ab></div>
+            </div>
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter6">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter6"><ab>ወሳድስ፡ ምስማክ፡</ab></div>
+              <div type="textpart" subtype="incipit"><ab>ወውእቱ፡ ለአይ፡ ግብር፡ ይሴኒ ወለኣይ፡ ገጽ፡ መክፈልተ፡ ጥበብ፡ ዘይትፈቀድ።</ab></div>
+              <div type="textpart" subtype="explicit"><ab>ወናሁ፡ ተበየነ፡ ምስማከ፡ ዝንቱ፡ መጽሓፍ፡ ወለአይ፡ ግብር፡ ይኄይስ።</ab></div>
+            </div>
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Chapter7">
+              <label>Preface</label>
+              <div type="textpart" subtype="title" xml:id="TitleChapter7"><ab>ወሳበዕ፡ አርእስተ፡ ነገር፡</ab></div>
```

Comment:
ወሳብዕ፡

Reply:
thank you for correcting all these typos, Massimo